nengo/networks/ensemblearray.py

```diff
@@ -89,6 +89,10 @@ def __init__(self, n_neurons, n_ensembles, ens_dimensions=1,
 
         super(EnsembleArray, self).__init__(label, seed, add_to_container)
 
+        for param in ens_kwargs:
+            if isinstance(ens_kwargs[param], np.ndarray):
+                ens_kwargs[param] = nengo.dists.Samples(ens_kwargs[param])
```

Comment:
> if people are accessing ensemble.encoders

How common is that? I think I never set `encoders` to something different than a distribution.

> I see the benefit of this distribution for fixing this specific problem

How feel you about leaving this block out of the PR? It requires a little bit more boilerplate on the user side, but also makes explicit what's happening. Also with this change `ensemble_array.ensembles[i].encoders` will never return a NumPy array. So if we worry about that in general, why not in the case of ensemble arrays?